src/boot/boot.js

```diff
@@ -197,17 +201,23 @@ export function bootHypothesisClient(doc, config) {
 
   const polyfills = polyfillBundles(commonPolyfills);
 
+  // Override styling in certain document viewers.
+  const viewerStyles = [];
+  if (
+    /** @type {MaybePDFWindow} */ (window).PDFViewerApplication !== undefined
+  ) {
+    viewerStyles.push('styles/pdfjs-overrides.css');
+  }
+
   injectAssets(
     doc,
     config,
     [
-      // Vendor code and polyfills
       ...polyfills,
+      ...viewerStyles,
 
       'scripts/annotator.bundle.js',
-
       'styles/highlights.css',
-      'styles/pdfjs-overrides.css',
```

Comment:
We are changing the order of the style bundles here. That shouldn't cause any problems, and I think the new order arguably makes more sense.